Classes/Services/MetsExporter.php

```diff
@@ -316,6 +316,28 @@ public function parseXPath($xPath)
         return $xml;
     }
 
+    public function sortPredicates($xpath) {
```

Comment:
Minor: The method is not "sorting" but prepending all non attribute related conditions, thus appending all conditions not starting with "@".